plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -56,6 +56,7 @@
 import io.trino.spi.type.Int128;
 import io.trino.spi.type.IntegerType;
 import io.trino.spi.type.NamedTypeSignature;
+import io.trino.spi.type.RealType;
```

Comment:
Remove unused import.

Reply:
Already addressed your comments in the latest commit, many thanks !